src/matcha_ml/state/matcha_state.py

```diff
@@ -4,14 +4,25 @@
 import hashlib
 import json
 import os
+import uuid
+from collections import defaultdict
 from dataclasses import dataclass
-from typing import Dict, List, Optional
+from typing import Dict, List, Optional, Tuple
 
 from matcha_ml.constants import MATCHA_STATE_PATH
-from matcha_ml.errors import MatchaError
+from matcha_ml.errors import MatchaError, MatchaInputError
 
 MISSING_STATE_ERROR_MSG = "No state file exists, you need to 'provision' resources or 'get' from already provisioned resources."
```

Comment:
During the `get` command we do pull from the cloud so it is technically correct but I agree it could lead to confusion. I will have a think about this.